lib/instream/connection.ex

```diff
@@ -132,6 +132,16 @@ defmodule Instream.Connection do
           _ -> QueryRunnerV1.write(points, opts, __MODULE__)
         end
       end
+
+      @impl Connection
+      def delete(points, opts \\ [])
+
+      def delete(points, opts) when is_map(points) do
```

Comment:
```suggestion
      def delete(points, opts \\ []) when is_map(points) do
```

I would suggest to merge both function heads as long as there is only one with an implementation.

Reply:
Thank you. Fixed